src/main/java/seedu/duke/command/AddCommand.java

```diff
@@ -22,14 +25,26 @@
 
     public static final String COMMAND_USAGE_NOTE = COMMAND_WORD_NOTE + ": Adds a note to notebook. Parameters: "
             + PREFIX_DELIMITER + PREFIX_TITLE + " TITLE "
-            + "[" + PREFIX_DELIMITER + PREFIX_TAG + " TAGS] "
+            + "[" + PREFIX_DELIMITER + PREFIX_TAG + " TAG TAG1 TAG2...] "
```

Comment:
The person has to enter the prefix for every tag they want to add so I think can leave it as "TAG" to avoid confusion.

Reply:
Because I fear if we do just "TAG" the person might assume it is only for 1 tag. What I can do is `PREFIX_TAG + " TAG " + PREFIX_TAG + " TAG "...` to show that they can add multiple tags? Would that be better?

Reply:
Thats also viable, also can add in "TAG COLOR" too

Reply:
addressed!